bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -39,21 +39,24 @@ spec:
       name: hardwareevents.event.redhat-cne.org
       version: v1alpha1
   description: |
+    ## Funtionality
+    Install and manage a hardware event delivery framework.
+
     ## Installation
-     Install and manage an Hardware event framework. This Operator needs
-    a secret to be created to access hardware event via redfish.
-    The secret  name and  spec  the operand expects are in the following format under the same namespace
-    the operand is deployed
+    This Operator needs a secret to be created to access Redfish
```

Comment:
You should start with CR I guess
 To start using  Hardware event proxy, install the operator and then create the following `HardwareEvent`:
    ```
    apiVersion: "event.redhat-cne.org/v1alpha1",
    kind: "HardwareEvent",
    metadata:
         name: "hardwareevent"
         spec:
             transportHost: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local"
    ```
Explain what is transport host.

Reply:
ignore this (all these changes should be done is base)